content/blog/2020-workshop-devsprints-tickets.md

```diff
@@ -0,0 +1,37 @@
+Title: Workshop and Devsprint tickets are now open
+Date: 2020-09-15 21:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-workshop-devsprint-tickets
+Summary: PyCon India 2020 workshop and devsprint tickets are now open.
+
+
+We know you all have been waiting for this eagerly. Don't worry, you don't have to wait anymore! The Workshop and devsprint tickets for PyCon India are now open. 
+
+[**Go get your tickets, available now on our website**](https://in.pycon.org/2020/#ticket)
+
+
+## Devsprints
+
+PyCon India 2020 Devsprints will happen on 4th and 5th October. We are happy to announce the devsprint tickets are now open. 
+
+You don't need to have a conference ticket to participate in the devsprints, anyone can buy the tickets. **The devsprint tickets will be free this year.**
+
+To know more about devsprints, check out our [blog on devsprints](). You can find the list of projects on our [wiki page](https://github.com/pythonindia/inpycon2020/wiki/List-of-PyCon-India-2020-Projects). If you are a contributor/maintainer for a project and would like to represent your project in the devsprints, please refer to our [devsprints page](https://github.com/pythonindia/inpycon2020/wiki/PyCon-India-2020-Devsprint) to add your project to our devsprints.
```

Comment:
Need the blog link here, but it's not been merged yet

Reply:
You can prefill the link using the slug in that unmerged article, or remove the reference altogether.

To that effect, will PR merge in before this gets in?